src/main/java/com/google/step/snippet/data/Card.java

```diff
@@ -28,4 +28,23 @@ public String getLink() {
   public String getDescription() {
     return description;
   }
+
+  @Override
+  public boolean equals(Object obj) {
+    if (this == obj) {
+      return true;
+    }
+    if (obj == null) {
+      return false;
+    }
+    if (!Card.class.isAssignableFrom(obj.getClass())) {
+      return false;
+    }
+    final Card other = (Card) obj;
+
```

Comment:
Super nit: I think it makes more sense to have a new line before the `final Card other` line and no new line between `final Card other` and the `return` statement.